pcdsdevices/beam_stats.py

```diff
@@ -9,6 +9,9 @@ class BeamStats(Device):
     ev = Cpt(EpicsSignalRO, 'SIOC:SYS0:ML00:AO541')
     rate = Cpt(EpicsSignalRO, 'EVNT:SYS0:1:LCLSBEAMRATE')
     owner = Cpt(EpicsSignalRO, 'ECS:SYS0:0:BEAM_OWNER_ID')
+    # This one is much faster than ev
+    accel_ev = Cpt(EpicsSignalRO, 'BLD:SYS0:500:PHOTONENERGY')
```

Comment:
This should have a more descriptive name, since both `ev` PVs come from the accelerator. Perhaps something like `ev_fast` or `fast_ev` or `ev_120`, etc.

I was waffling on whether or not we should have both PVs, it might make sense to just replace the other `ev` measurement with this faster one. The old one is like 2/3Hz, and this one is 120Hz.

@teddyrendahl any opinion about this?

Reply:
Are they calculated differently? Is there any reason to compare the two? I thought `SIOC` was just an estimate maybe the `BLD` is more of a systematic measurement. The fact that it is fact makes it sound to me like it is being triggered by something?